azure-common/src/main/java/com/azure/common/configuration/InvalidConfigurationException.java

```diff
@@ -0,0 +1,15 @@
+package com.azure.common.configuration;
+
+/**
+ * Exception thrown when the {@link ClientConfiguration} is not valid for use with a
+ * {@link com.azure.common.ServiceClient}.
+ */
+public class InvalidConfigurationException extends RuntimeException {
+    /**
+     * Creates an instance with the given message.
+     * @param message String describing why the configuration is invalid.
+     */
+    public InvalidConfigurationException(String message) {
+        super(message);
+    }
+}
```

Comment:
If you aren't going to add any additional context information into this class, I would remove it and use IllegalStateException instead. Soon I will be speccing up the list of common exceptions we will use across the SDK, and ISE is one of them.

Reply:
Gotcha. I'll remove this.